src/plugins/dashboard/public/services/application/application_service.ts

```diff
@@ -22,7 +22,7 @@ export const applicationServiceFactory: ApplicationServiceFactory = ({ coreStart
       navigateToApp,
       navigateToUrl,
       getUrlForApp,
-      capabilities: { advancedSettings, maps, navLinks, visualize },
+      capabilities: { advancedSettings, maps, navLinks, visualize, savedObjectsManagement },
```

Comment:
The `savedObjectsManagement` capabilities need to be passed to the `TableListViewTable` component to enable or disable the share to space flyout.